namespace-provisioner/about.hbs.md

```diff
@@ -16,53 +16,50 @@ Use Namespace Provisioner with one of the following modes:
 Controller mode
 : Controller mode has the following characteristics:
 
+    ![Diagram of .](../images/namespace-provisioner-controller.png)
+
     - The list of developer namespaces is managed by the Namespace Provisioner controller using a label selector `apps.tanzu.vmware.com/tap-ns=""`
-    - Namespace Provisioner creates default resources that are shipped Out of the Box in all managed namespaces.
-    - Namespace Provisioner creates additional Platform Operator templated resources stored in Git repository locations specified under the `additional_sources` section in Namespace Provisioner configuration. For more information, see [Install Namespace Provisioner](customize-installation.md).
+    - Namespace Provisioner creates [default resources](reference.md#default-resources) that are shipped Out of the Box in all managed namespaces.
```

Comment:
This does not link anywhere, the text appears as it's written, I guess it's because it's inside a code block.